src/Microsoft.DotNet.Interactive.Formatting/DefaultHtmlFormatterSet.cs

```diff
@@ -155,6 +155,13 @@ internal class DefaultHtmlFormatterSet
                     return formatter.Format(context, value, writer);
                 }),
 
+                // BigInteger should be displayed as plain text
+                new HtmlFormatter<System.Numerics.BigInteger>((context, value, writer) =>
+                {
+                    HtmlFormatter.FormatStringAsPlainText(value.ToString(), writer);
```

Comment:
I did it same way it's done for strings, but I'm not sure it should work like this.

Reply:
Calling `ToString` here will bypass the `PlainTextFormatter`, which might have its own customization in place. To make sure the behavior is consistent, you can call into the other formatter like this:

```cs
value.FormatTo(writer, PlainTextFormatter.MimeType);
```

With that in mind, a complete fix should include a default for that formatter as well, otherwise:

![image](https://user-images.githubusercontent.com/547415/114777875-07804980-9d29-11eb-8a0a-bc0878c057a5.png)